tests/test_lca.py

```diff
@@ -399,12 +399,21 @@ def test_databases_load_fail_on_dir():
     with pytest.raises(ValueError) as exc:
         dblist, ksize, scaled = lca_utils.load_databases([filename1])
 
+    err = filename1.err
```

Comment:
here, 'filename' is a string. You probably want this to be `err = str(exc.value)` or something.